docs/alternative-environment-tips/docksal.md

```diff
@@ -0,0 +1,94 @@
+# Setting up BLT with Docksal
+
+By default, BLT is set to work with Drupal VM. It is easy to set up to use with Docksal, but
+you will have to install the Docksal `blt` addon command and make some configuration changes.
+
+## Setting Your Project to Use Docksal
+
+If you already have Docksal installed on your system, setting your BLT project to use Docksal only
+requires that you add a `.docksal` directory in your project and run `fin up` in the command prompt
+of your project directory. See the [Docksal Documentation](https://docs.docksal.io) for more specific
+Docksal information.
+
+## Using the Docksal `blt` Command
+
+BLT will install a `blt alias` command on your host system. Running the `blt` command will
+then use the resources of your host system. Instead, you need to make sure you're running
+the commands through the Docksal CLI service.
+
+Install the Docksal `blt` addon command:
+```
+fin addon install blt
+```
+
+Run any blt command with `fin` and you will be running the command within the Docksal CLI, e.g.:
+```
+fin blt doctor
+```
+
+
+## Setting the Database Connection and Drush Alias
+
+### New Project Setup
+
+The default Docksal database settings are different than the BLT defaults. If you have not already 
+added special settings to the local.settings.php file that BLT creates, it is best to begin by 
+following these steps:
+
+1. remove the `docroot/sites/default/settings/local.settings.php` and the `docroot/sites/default/local.drush.yml` files
+2. update the blt.yml file in your project's blt directory:
+
+    ```
+    project:
+      machine_name: myproject
+      local:
+        hostname: '${env.VIRTUAL_HOST}'
+    drupal:
+      db:
+        database: default
+        username: user
+        password: user
+        host: db
+        port: 3306
+    ```
+3. initiate and verify blt settings
+    ```
+    fin blt blt:init:settings
+    ```
+4. setup your Drupal site
+    ```
+    fin blt setup -D setup.strategy=install
```

Comment:
I would elaborate on what `setup.strategy` means here. By default, passing this option is not required since BLT defaults to the install strategy.

Maybe specify something along the lines of:

Alternatively, you may use a sync setup strategy if there is an existing remote with a source database to used in a Drush sql:sync operation:
```
fin blt setup -D setup.strategy=sync
```